data2rdf/parsers/csv.py

```diff
@@ -120,10 +120,16 @@ class CSVABoxParser(ABoxBaseParser):
     )
     metadata_length: int = Field(..., description="Length of the metadata")
     dataframe_sep: Optional[str] = Field(
-        None, description="Column separator of the time series header"
+        None,
+        description="Column separator of the time series header",
+        alias=AliasChoices("dataframe_sep", "time_series_sep"),
     )
     dataframe_header_length: int = Field(
-        2, description="Length of header of the time series"
+        2,
+        description="Length of header of the time series",
```

Comment:
Also here

Reply:
"time series" has been replaced globally in all descriptions within the package.